billboard.java

```diff
@@ -0,0 +1,56 @@
+//package usaco2018january;
+//All 10 tests work no errors
+//50 min to complete
+//Difficulty may have been a 7/10
+//I thought I did okay but I didn't read the question cafefully and wasted 20 minutes overthinking the question
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Scanner;
+import java.util.StringTokenizer;
+
+public class billboard
+{
+
+
+	public static void main(String[] args) throws IOException
+	{
+		//----------------------------------------------------------------------------------------------------------------------------------------------------
+		BufferedReader br = new BufferedReader(new FileReader("billboard.in"));
+		PrintWriter pw = new PrintWriter(new BufferedWriter(new FileWriter("billboard.out")));
+        //----------------------------------------------------------------------------------------------------------------------------------------------------------------
+		StringTokenizer st = new StringTokenizer(br.readLine());
+		int MowerX1=Integer.parseInt(st.nextToken());
+	    int MowerY1=Integer.parseInt(st.nextToken());
+	    int MowerX2=Integer.parseInt(st.nextToken());
+	    int MowerY2=Integer.parseInt(st.nextToken());        
+        st = new StringTokenizer(br.readLine());
+        int CowFeedX1=Integer.parseInt(st.nextToken());
+        int CowFeedY1=Integer.parseInt(st.nextToken());
+        int CowFeedX2=Integer.parseInt(st.nextToken());
+        int CowFeedY2=Integer.parseInt(st.nextToken());
+        //-------------------------------------------------------------------------------------------------------------------------------------------------
+        

```

Comment:
Comment to your three cases. When debugging, the top level logic is not easy to read by looking at the code, comment so you know what you are checking.